app/assets/stylesheets/administrate/library/_variables.scss

```diff
@@ -12,7 +12,7 @@ $base-line-height: 1.5;
 $heading-line-height: 1.2;
 
 // Other Sizes
-$modular-scale-ratio: $minor-third;
+$modular-scale-ratio: 1.2;
```

Comment:
This is a Bourbon setting for it’s `modular-scale` function. If that function is not used anywhere, this variable should be able to be removed.